### PR TITLE
[Hack Update] 007-AzureMonitoring V2 Refresh 2023

### PR DESCRIPTION
**Pull request by jrzyshr**
This is the long-awaited refresh of the Azure Monitoring What The Hack!

The hack has been re-written from start to finish.  This PR is essentially a new hack.

The old hack's challenge scenario is still in place.  Improvements include:
- New WTH templates used for Student & Coach guides
- New Bicep templates authored to deploy the initial Azure environment.
- New challenge on Azure Monitor for Containers with AKS
- Updated lecture deck
- Updated to use latest Azure Monitor features.

### CODE REVIEW

**Review by perktime — CHANGES_REQUESTED** [review]

File: 007-AzureMonitoring/Student/Challenge-00.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+## Learning Resources
+
+- [Get Started with Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/get-started-azureps?view=azps-6.4.0)
+- [Get Started with Azure Command-Line Interface (CLI)](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli)
+- [Overview of Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+
+## Tips
+ 
+- Make sure you have the compute capacity in the region you are deploying to and request an increase to the limit if needed.
+- Make sure you are using a region that supports the public preview for Azure Monitor for VMs
```
[the remark this answers is not in the document]
Comment (jrzyshr) [reply 007-AzureMonitoring/Student/Challenge-00.md]: Done.  Removed the outdated tip.

File: 007-AzureMonitoring/Coach/Solution-02.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+- At the top you will see the Resource ID that will have the following structure:  
+`/subscriptions/<sub-id>/resourceGroups/<rg-name>/providers/microsoft.insights/actionGroups/<action-group-name> `
+  
+
+### Test Activity log alert
+
+- Login to Azure portal and stop you Visual Studio VM by clicking the "Stop" button.  
+
+![Stopping the  Virtual Machine via “Stop” button in Azure Portal](https://github.com/msghaleb/AzureMonitorHackathon/raw/master/images/stopVM.png)
```
Comment (Whowong) [inline 007-AzureMonitoring/Coach/Solution-02.md:24]: Commenting here but I see them in a number of solution files.  Will the links to the images be changed to the local files?  I see some pointing to local images and others pointing to another repo.

File: 007-AzureMonitoring/Coach/Solution-01.md
Comment (Whowong) [inline 007-AzureMonitoring/Coach/Solution-01.md:1]: I see that it was mentioned the first challenge can be challenging for some students.  Have we considered breaking it up into multiple smaller challenges?

File: 007-AzureMonitoring/Coach/Solution-02.md (hunk 2 above)
Comment (jrzyshr) [reply 007-AzureMonitoring/Coach/Solution-02.md:24]: This needs to be fixed/udpated!

File: 007-AzureMonitoring/Coach/Solution-01.md
Comment (jrzyshr) [reply 007-AzureMonitoring/Coach/Solution-01.md:1]: This was discussed & debated at multiple points during development, we put it into one challenge as all of the tasks were working towards the same goal.  We could consider breaking it up in a future update based on feedback from any deliveries, but I vote to leave it as is for now.

Comment (Whowong) [reply 007-AzureMonitoring/Coach/Solution-01.md:1]: Sounds good, thanks for the context.

File: 007-AzureMonitoring/Coach/Solution-02.md (hunk 2 above)
Comment (jrzyshr) [reply 007-AzureMonitoring/Coach/Solution-02.md]: Done!

**Review by perktime — APPROVED** [review]

**Pull request merged**